build.sbt

```diff
@@ -2,7 +2,7 @@ import MimaSettings.mimaSettings
 
 publish / skip := true
 
-def Scala3 = "3.3.3"
+def Scala3 = "3.4.1"
```

Comment:
```suggestion
def Scala3 = "3.3.3"
```

Reply:
I don't know why this change affects the scripted tests but reverted it anyway

Reply:
https://github.com/scalikejdbc/scalikejdbc/actions/runs/9040915213/job/24845723955

Reply:
Scala 3 has own version info in `.tasty` file unlike Scala 2.

https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html

> Unless your library is built around some new language feature available only on Next, you should stick to the latest LTS version to have the broadest possible user base.